scripts/cover/coverGcov.sh

```diff
@@ -1,50 +1,46 @@
 #!/usr/bin/env bash
 set -e
-
 # set HOOT_HOME if not set
 if [ -z "$HOOT_HOME" ]; then
     HOOT_HOME=~/hoot
 fi
-
 # generate gcov files from gcda files
 cd $HOOT_HOME
-
+# create the gcov output folder
+if [ ! -d gcov ]; then
+    mkdir gcov
+else
+    rm -f gcov/*
+fi
 # hoot-core coverage
-mkdir gcovCore
-cd gcovCore
-gcov ../hoot-core/tmp/debug/*.gcda
+pushd hoot-core
```

Comment:
`pushd` pushes a directory on a stack and changes to the directory.  The subsequent `popd` returns to the previous directory and pops the directory off of the stack.